### PR TITLE
fix compilation from swift wrapping

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
Small fix which appears by using DGtal in swift based application.

# Checklist

- [ ] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [ ] Doxygen documentation of the code completed (classes, methods, types, members...)
- [ ] Documentation module page added or updated.
- [ ] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [ ] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [ ] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

File: src/DGtal/base/Trace.ih [hunk 1]
```diff
@@ -259,7 +259,7 @@ DGtal::Trace::progressBar(const double currentValue, const double maximumValue)
     }
   // and back to line begin - do not forget the fflush to avoid output buffering problems!
 
-  BOOST_STATIC_CONSTANT(std::string, rotation_string = "|\\-/");
+  std::string rotation_string = "|\\-/";
```
Comment (dcoeurjo) [inline src/DGtal/base/Trace.ih]: const std::string ... ?

(I agree that the static stuff is not necessary)

Comment (kerautret) [reply src/DGtal/base/Trace.ih]: yes ;)

**Comment by dcoeurjo** [comment]
could you please merge the master ?

**Comment by dcoeurjo** [comment]
(+ changelog?)

**Comment by kerautret** [comment]
done ;)

**Comment by dcoeurjo** [comment]
thanks merging

**Pull request merged**